src/Microsoft.Xaml.Behaviors/Microsoft.Xaml.Behaviors.csproj

```diff
@@ -24,28 +24,21 @@
     <UseWPF>true</UseWPF>
     <GenerateNuspecDependsOn>$(GenerateNuspecDependsOn);TimestampNugetPackage</GenerateNuspecDependsOn>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net462|AnyCPU'">
+    <NoWarn>1701;1702;1591</NoWarn>
```

Comment:
Since the values are the same for all configs/TFMs how about just adding it to the unconditional PropertyGroup above?

Reply:
Good catch. That was autogenerated by VS.